seedsource_core/django/seedsource/report.py

```diff
@@ -85,7 +85,15 @@ def get_context_variables(self):
         is_imperial = self.configuration['unit'] == 'imperial'
 
         for variable in self.configuration['variables']:
-            name, value, transfer = variable['name'], variable['value'], variable['transfer']
+            name, transfer = variable['name'], variable['transfer']
+            value = (
+                variable["zoneCenter"]
+                if (
+                        self.configuration["method"] == "seedzone"
+                        and self.configuration["center"] == "zone"
+                )
+                else variable["value"]
+            )
```

Comment:
Given that this stretches to multiple lines anyway, I'd rather express it as a traditional `if/else`:

```python
if self.configuration["method"] ...:
    value = variable["zoneCenter"]
else:
    value = variable["value"]
```